scripts/create_scan_config.sh

```diff
@@ -17,8 +17,9 @@ IMAGE=${1}
 echo "Creating security scan configuration file:"
 cat <<EOF | tee ${FILENAME}
 module-name: btp-operator
+rc-tag: ${IMAGE}
```

Comment:
Shouldn't we rename it to different name if the meaning changes.

Reply:
I will rename it to `TAG`